test/fixtures/aircraftFixtures.js

```diff
@@ -13,3 +13,15 @@ const runwayAssignmentMock = airportModelFixture.getRunway('19L');
 export const fmsArrivalFixture = new Fms(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, runwayAssignmentMock, AIRCRAFT_DEFINITION_MOCK, navigationLibraryFixture);
 export const fmsDepartureFixture = new Fms(DEPARTURE_AIRCRAFT_INIT_PROPS_MOCK, runwayAssignmentMock, AIRCRAFT_DEFINITION_MOCK, navigationLibraryFixture);
 export const modeControllerFixture = new ModeController();
+
+export function createFmsArrivalFixture() {
+    return new Fms(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, runwayAssignmentMock, AIRCRAFT_DEFINITION_MOCK, navigationLibraryFixture);
+}
+
+export function createFmsDepartureFixture() {
+    return new Fms(DEPARTURE_AIRCRAFT_INIT_PROPS_MOCK, runwayAssignmentMock, AIRCRAFT_DEFINITION_MOCK, navigationLibraryFixture);
+}
+
+export function createModeControllerFixture() {
```

Comment:
why are you using functions here instead of just exporting an instance?
```javascript
export const modeControllerFixture = new ModeController();
```

Reply:
when i simpy export and import an instance, that instance can be mutated throughout subsequent tests in the same test file. I need the instance to be generated fresh each time. I could do that in the test file, but that would require a myriad of extra imports. Instead I just added a create-a-new-instance helper function that I can import and run to achieve the desired result.

Reply:
ok, that makes sense.  

_(there should probably be a comment explaining the why somewhere in that file 😉 )_

Reply:
Fair enough, done!